### DISCUSSION TITLE
Option number 2: Json and album

### DISCUSSION THREAD

**Discussion by chiccopeso** [TheLastGimbus/GooglePhotosTakeoutHelper#214] [I NEED HELP]
I selected the Json output option and i thought to find a single folder and album already made but it doesn't. Did I make a mistake?
I've tried all the option and no one give me album, only folders in some case (yeah, i could make the album from the folder but we are talking of 30+ albums)
Thanks

**Comment by TheLastGimbus** [comment]
What do you mean? You need to describe it better...

What album do you want? On you computer? Only thing the scrpit can make on the computer is folders...

**Reply by chiccopeso** [reply]
On Google photo i have a lot of pictures and some albums but a single copy of any photo. What i would like is to find inside Syn Photo the same structure (1 huge folder and some album but only 1 copy of each photo)  but I find folders relative to the album of G photo. If I want album i should create album starting by each folder, it is not a problem but it means that I will have 2 copy of the same photo (1 in the main folder and 1 in the folder from i create album) qnd if I erase the folder after made the album i will lose the album it self. I can make folders with photo linked to the original to save space but it is not the same as G photo

**Reply by TheLastGimbus** [reply]
Uuummm yyy... :thinking: 

You want a Synology Photos album? Then sorry but their albums is their thing

You want a single copy of photos to save space? Great! That's what `shortcut` option is for: https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/discussions/187#discussion-4980576

**Comment by chiccopeso** [comment]
Yeah i know about shortcut but if I create the album based on the shortcut
folder i can not erase the folder anymore or i would lose the album.
I'll figure it out how to do it, thanks

Il gio 8 giu 2023, 15:18 Mateusz Soszyński ***@***.***> ha
scritto:
